File: OPENAPI-REST-API/swagger-client/dart/lib/model/quickserver_ip_table_row.dart
part of swagger.api;

class QuickserverIpTableRow {
  /* Description */
  String desc = null;
/* Value */
  String value = null;

  QuickserverIpTableRow();

  @override
  String toString() {
    return 'QuickserverIpTableRow[desc=$desc, value=$value, ]';
  }

  QuickserverIpTableRow.fromJson(Map<String, dynamic> json) {
    if (json == null) return;
    desc = json['desc'];
    value = json['value'];
  }

  Map<String, dynamic> toJson() {
    return {
      'desc': desc,
      'value': value
     };
  }

  static List<QuickserverIpTableRow> listFromJson(List<dynamic> json) {
    return json == null ? new List<QuickserverIpTableRow>() : json.map((value) => new QuickserverIpTableRow.fromJson(value)).toList();
  }

  static Map<String, QuickserverIpTableRow> mapFromJson(Map<String, Map<String, dynamic>> json) {
    var map = new Map<String, QuickserverIpTableRow>();
    if (json != null && json.length > 0) {
      json.forEach((String key, Map<String, dynamic> value) => map[key] = new QuickserverIpTableRow.fromJson(value));
    }
    return map;
  }
}

File: OPENAPI-REST-API/swagger-client/csharp-dotnet2/src/main/CsharpDotNet2/myadmin-client-csharp-dotnet2/Model/QuickserverIpTableRow.cs
using System;
using System.Text;
using System.Collections;
using System.Collections.Generic;
using System.Runtime.Serialization;
using Newtonsoft.Json;

namespace IO.Swagger.Model {

  /// <summary>
  /// A single row in the QuickServer IP information table.
  /// </summary>
  [DataContract]
  public class QuickserverIpTableRow {
    /// <summary>
    /// Description
    /// </summary>
    /// <value>Description</value>
    [DataMember(Name="desc", EmitDefaultValue=false)]
    [JsonProperty(PropertyName = "desc")]
    public string Desc { get; set; }

    /// <summary>
    /// Value
    /// </summary>
    /// <value>Value</value>
    [DataMember(Name="value", EmitDefaultValue=false)]
    [JsonProperty(PropertyName = "value")]
    public string Value { get; set; }


    /// <summary>
    /// Get the string presentation of the object
    /// </summary>
    /// <returns>String presentation of the object</returns>
    public override string ToString()  {
      var sb = new StringBuilder();
      sb.Append("class QuickserverIpTableRow {\n");
      sb.Append("  Desc: ").Append(Desc).Append("\n");
      sb.Append("  Value: ").Append(Value).Append("\n");
      sb.Append("}\n");
      return sb.ToString();
    }

    /// <summary>
    /// Get the JSON string presentation of the object
    /// </summary>
    /// <returns>JSON string presentation of the object</returns>
    public string ToJson() {
      return JsonConvert.SerializeObject(this, Formatting.Indented);
    }

}
}
